databases/week3/dbweek3.sql

```diff
@@ -0,0 +1,261 @@
+   60.50, '2021-09-11', 5);
+
+-- Inserts to reservation table
+INSERT INTO
+  reservation(number_of_guests, created_date, contact_phonenumber, contact_name, meal_id)
+VALUES(3, '2021-10-11', '+ 45 50 000 00', 'John', 1);
+
+INSERT INTO
+  reservation(number_of_guests, created_date, contact_phonenumber, contact_name, meal_id)
+VALUES(5, '2021-11-11', '+ 45 50 001 00', 'Alex', 7);
+
+INSERT INTO
+  reservation(number_of_guests, created_date, contact_phonenumber, contact_name, meal_id)
+VALUES(2, '2021-11-12', '+ 45 50 001 10', 'Maria', 5);
+
+INSERT INTO
+  reservation(number_of_guests, created_date, contact_phonenumber, contact_name, meal_id)
+VALUES(4, '2021-11-13', '+ 45 50 001 11', 'Natalia', 6);
+
+-- Inserts to review table
+INSERT INTO
+  review(title, review_meal_id, created_date, stars)
+VALUES('Review for carbonara', 1 , '2021-08-12', 3 );
+INSERT INTO
+  review(title, review_meal_id, created_date, stars)
+VALUES('Review avocado toast', 5 , '2021-09-16', 7 );
+INSERT INTO
+  review(title, review_meal_id, created_date, stars)
+VALUES('Review for Greek salat', 7 , '2021-10-12', 9 );
+INSERT INTO
+  review(title, review_meal_id, created_date, stars)
+VALUES('Review for vanill cake', 8 , '2021-08-19', 1 );
+
+-- Meal - Queries to write
+-- Get all meals
+SELECT 
+  title
+FROM 
+  meal;
+
+-- Get a meal with any id, fx 1
+SELECT 
+  title
+FROM meal
+WHERE id = 2;
+
+-- Update a meal with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE
+	meal 
+SET
+	price  = 150 
+WHERE
+	id = 1; 
+
+-- Delete a meal with any id, fx 1
+DELETE FROM 
+  meal
+WHERE 
+  id = 2; 
+
+-- Reservation - Queries to write
+-- Get all reservations
+SELECT 
+  *
+FROM
+  reservation;
+
+-- Get a reservation with any id, fx 1
+SELECT 
+  created_date
+FROM reservation
+WHERE id = 1;
+
+-- Update a reservation with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE
+	reservation 
+SET
+	number_of_guests = 4 
+WHERE
+	id = 1; 
+
+-- Delete a reservation with any id, fx 1
+DELETE FROM 
+  reservation
+WHERE 
+  id = 12; 
+
+-- Review - Queries to write
+-- Get all reviews
+SELECT
+  *
+FROM
+  review;
+
+-- Get a review with any id, fx 1
+SELECT 
+  *
+FROM 
+  review
+WHERE id = 1;
+
+-- Update a review with any id, fx 1. Update any attribute fx the title or multiple attributes
+UPDATE
+	review 
+SET
+	stars = 5 
+WHERE
+	id = 1; 
+
+-- Delete a review with any id, fx 1
+DELETE FROM 
+  reservation
+WHERE 
+  id = 1; 
+
+-- Additional queries
+-- Functionality
+-- Get meals that has a price smaller than a specific price fx 90
+SELECT
+ *
+FROM
+  meal
+WHERE price < 90;
+
+-- Get meals that still has available reservations
+SELECT
+  meal.title, meal.max_reservations, COUNT(reservation.meal_id) as reservation_amount
+FROM
+  meal
+JOIN 
+  reservation 
+ON meal.id = reservation.meal_id
```

Comment:
would avoid breaking lines on FROM and JOIN statements, improves readability and clarity.